pom.xml

```diff
@@ -142,7 +142,8 @@ LICENSE file.
     <redis.version>2.9.0</redis.version>
     <riak.version>2.0.5</riak.version>
     <rocksdb.version>6.2.2</rocksdb.version>
-    <s3.version>1.10.20</s3.version>
+    <s3.version>1.11.820</s3.version>
```

Comment:
put the s3 update in its own PR?

Reply:
My mistake. Removed.

btw: the S3 implementation seems wrong though.